test/serialized_attributes_test.rb

```diff
@@ -86,9 +88,9 @@ def setup
     end
 
     test "#attribute_names contains serialized fields" do
-      assert_equal %w(active age average birthday extras lottery_picks names title), @record.attribute_names
+      assert_equal %w(active age average birthday extras id lottery_picks names title), @record.attribute_names
```

Comment:
This appears to be a recent change in ActiveRecord - the stock `#attribute_names` now includes the `id` column